pkg/codegen/internal/test/testdata/simple-resource-schema/go/example/resource.go

```diff
@@ -23,6 +23,14 @@ func NewResource(ctx *pulumi.Context,
 		args = &ResourceArgs{}
 	}
 
+	// Always mark these fields as secret to avoid leaking sensitive values into the state.
+	if args.Bar != nil {
+		args.Bar = pulumi.ToSecret(args.Bar).(pulumi.StringInput)
```

Comment:
Does this work?  I would have expected this to need to be `StringOutput`.

Reply:
It does work, although it looks like `StringOutput` works as well, so I'll change to that. I recall having problems with the Output type assertions when I was figuring this out initially, but it must have been something else going on.